l10n_it_fatturapa_in/wizard/wizard_import_fatturapa.py

```diff
@@ -850,6 +850,8 @@ def invoiceCreate(
                 comment += rel_doc + '\n'
 
         invoice_data = {
+            'date': fatturapa_attachment.e_invoice_received_date or
```

Comment:
Queste righe direi sono quelle incriminate, che vanno a pre-compilare il campo 'date' che è il campo di competenza contabile della registrazione, con la data di registrazione ed è sbagliato. La registrazione contabile della fattura (da cui si basano poi i calcoli dell'IVA) si basa sulla data della fattura ricevuta.

Reply:
Ok! Ho appena pushato la modifica con l'aggiunta del campo in questione su ``account.invoice``